src/x/ident/types.go

```diff
@@ -241,6 +241,11 @@ func NewTags(values ...Tag) Tags {
 	return Tags{values: values}
 }
 
+// Reset resets the tags for reuse.
+func (t *Tags) Reset(values []Tag) {
+	t.values = values
```

Comment:
Does this need to take ownership of the tags?

Reply:
Followed up, ident.ID does the ref tracking so we are ok here.